stetho/src/main/java/com/facebook/stetho/inspector/elements/android/TextViewDescriptor.java

```diff
@@ -5,21 +5,30 @@
 import android.text.Editable;
 import android.text.TextWatcher;
 import android.widget.TextView;
-
 import com.facebook.stetho.common.Util;
 import com.facebook.stetho.inspector.elements.AttributeAccumulator;
 import com.facebook.stetho.inspector.elements.ChainedDescriptor;
 
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.IdentityHashMap;
 import java.util.Map;
 
 final class TextViewDescriptor extends ChainedDescriptor<TextView> {
   private static final String TEXT_ATTRIBUTE_NAME = "text";
 
+  private final MethodInvoker mMethodInvoker;
```

Comment:
I implemented this based on @jasta's comment above: `Why only support android:text? The system we have defines something much more general that should be able to set any field we define. Perhaps what's missing is a mapping function or a field/method name for reflection. Then you could ask any descriptor to assign any attribute it emitted.`.